lib/handler_stack.coffee

```diff
@@ -65,7 +65,11 @@ class HandlerStack
           true # Do nothing, but continue bubbling.
         else
           # result is @suppressEvent or falsy.
-          DomUtils.suppressEvent event if @isChromeEvent event
+          if @isChromeEvent event
+            if type == "keydown"
+              DomUtils.consumeKeyup event
+            else
+              DomUtils.suppressEvent event
```

Comment:
Help me understand this.

Here, lines 68 to 72, is the key idea of this PR.  We take the responsibility of deciding what to do with `keyup` out of higher-level modes, and into the handler stack.  If a `keydown` event was handled by Vimium, then the corresponding `keyup` event is blocked.  Otherwise it isn't.

That's it, yes?